avalon/tools/models.py

```diff
@@ -62,7 +62,8 @@ def setData(self, index, value, role=QtCore.Qt.EditRole):
                 item[key] = value
 
                 # passing `list()` for PyQt5 (see PYSIDE-462)
-                self.dataChanged.emit(index, index, list())
+                args = () if Qt.IsPySide or Qt.IsPyQt4 else ([],)
```

Comment:
There should be role in the list. `args = () if Qt.IsPySide or Qt.IsPyQt4 else ([role],)`

Reply:
Yeah, I think you are right !